decidim/decidim
# Removal of unused show pages in admin components

#### :tophat: What? Why?
Upon working on https://github.com/decidim/decidim/pull/10976 and https://github.com/decidim/decidim/pull/11116, I noticed some unused show pages in the components under Admin. 

This PR is to remove these unused pages in the develop branch. 

#### :pushpin: Related Issues
- Fixes #11140 

#### Testing

1. Head over to these pages in the develop branch: 
```
    decidim-admin/app/views/decidim/admin/attachment_collections/show.html.erb
    decidim-admin/app/views/decidim/admin/attachments/show.html.erb
    decidim-admin/app/views/decidim/admin/categories/show.html.erb
```
2. See they have been deleted.
3. Try to access a ``` attachment_collections/show.html.erb ``` within Votings (Elections) by deleting the "Edit" in the browser. Notice its accessible. 

### :camera: Screenshots

:hearts: Thank you!
